metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/NetworkFunctions.java

```diff
@@ -163,12 +163,16 @@ public Object apply(List<Object> objects) {
   public static class URLToPort extends BaseStellarFunction {
     @Override
     public Object apply(List<Object> objects) {
```

Comment:
I did make these changes to the variables, but did not change the docs since the functions are literally called `URL_*`.